app/src/main/res/layout/emoji_category_layout.xml

```diff
@@ -15,4 +15,14 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+        android:id="@+id/tv_recent_emoji"
```

Comment:
Change the `tv_recent_emoji` to `text_no_recent_emoji`. This will be more meaningful (and will follow our code style).